manual/src/refman/lex.etex

```diff
@@ -302,16 +348,16 @@ longest first token.
 
 \begin{syntax}
 linenum-directive:
-     '#' {{"0" \ldots "9"}} '"' { string-character } '"'
+        '#' {{"0" \ldots "9"}}
+      | '#' {{"0" \ldots "9"}} '"' { string-character } '"'
```

Comment:
This is probably an unintended mistake: you are reverting recent changes that fixed the line directive grammar and description.

Reply:
Thanks. I will look through the history of these files and make sure all recent changes are taken into account.

Reply:
Fixed.